mkosi/resources/mkosi-obs/mkosi.postoutput

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# End of first stage of build:
+# - built UKI is in $OUTPUTDIR
+# - get PCR policy digests if any, or PE hash(es) with pesign
+# - pack them up in a cpio as OBS expects and store them in OTHER/
+# - create minimal recipe for second stage that will continue from here
+
+set -e
+
+declare -a UNSIGNED
+UNSIGNED=( "$(find "$OUTPUTDIR" -type f \( -name "*.efi" -o -name "vmlinu*" \) -printf '%P\n')" )
+
+if [ "${#UNSIGNED[@]}" -eq 0 ]; then
+    echo "No unsigned files found, exiting"
+    exit 0
+fi
+
+nss_db="$PWD/nss-db"
+# certutil will fail if it's called twice
+rm -rf "$nss_db"
+mkdir -p "$nss_db" hashes
+certutil -N -d sql:"$nss_db" --empty-password
+
+for f in "${UNSIGNED[@]}"; do
+    if [ -f "${OUTPUTDIR}/${f%.efi}.pcrs" ]; then
+        mkdir -p "hashes/pcrs/$f"
+        while read -r pol; do
+            echo -n "$pol" | tr '[:lower:]' '[:upper:]' | basenc --base16 --decode >"hashes/pcrs/${f}/${pol}"
+        done < <(jq -r 'to_entries[] | .value[].pol' <"${OUTPUTDIR}/${f%.efi}.pcrs")
```

Comment:
haven't checked this with actual data yet, but maybe the whole thing could be
```suggestion
        jq -r 'to_entries[] | .value[].pol | ascii_downcase' "${OUTPUTDIR}/${f%.efi}.pcrs" | 
            basenc --base16 --decode >"hashes/pcrs/${f}/${pol}"
```

Reply:
$pol is not defined in that case